timemachine/potentials/nonbonded.py

```diff
@@ -235,7 +235,7 @@ def nonbonded(
     # funny enough lim_{x->0} erfc(x)/x = 0
     eij_charge = jnp.where(keep_mask, qij * erfc(beta * dij) * inv_dij, 0)  # zero out diagonals
     if cutoff is not None:
-        eij_charge = jnp.where(dij > cutoff, 0, eij_charge)
+        eij_charge = jnp.where(dij < cutoff, eij_charge, 0)
```

Comment:
This is the change in question. Per https://github.com/proteneer/timemachine/blob/master/timemachine/cpp/src/kernels/k_nonbonded.cuh#L367-L369 this should be strictly less than. 

Other changes are just some minor things I noticed while investigated the nonbonded potentials.